HttpStatusCodes_C++11.h

```diff
@@ -111,6 +111,13 @@ enum class Code
 	NetworkAuthenticationRequired = 511  //!< Indicates that the client needs to authenticate to gain network access.
 };
 
+/*! Explicitly converts code to int
```

Comment:
The parameter and return value should be formally documented:
```suggestion
/*! Converts a Code to its corresponding integer value.
 * \param code The code to be converted.
 * \return The numeric value of \p code.
```